include/sta/ConcreteLibrary.hh

```diff
@@ -105,6 +106,8 @@ public:
   ConcreteCellPortBitIterator *portBitIterator() const;
   bool isLeaf() const { return is_leaf_; }
   void setIsLeaf(bool is_leaf);
+  void setAttribute(const char *key, const char *value);
```

Comment:
The formatting convention is for each function argument to be on a separate line.
This comment applies to most of the multi-arg functions defined in the PR.